app/root.tsx

```diff
@@ -98,6 +98,7 @@ export function Dokument({ children, språk = ELocaleType.NB }: DokumentProps) {
   return (
     <html lang={språk}>
       <head>
+        <title>Endringsmelding</title>
         <meta charSet="utf-8" />
```

Comment:
Mer dokumentasjon om dette [her](https://www.w3schools.com/html/html_head.asp).
Mest for SEO, men det står også at det er required i html.